Subproject2/3. Data Layer/Database Context/SOVAContext.cs

```diff
@@ -116,6 +115,9 @@ protected override void OnModelCreating(ModelBuilder modelBuilder)
             modelBuilder.Entity<Submission>().HasOne(a => a.SoMember).WithMany().HasForeignKey(a => a.SoMemberId);
             modelBuilder.Entity<UserHistory>().HasOne(a => a.User).WithMany().HasForeignKey(a => a.UserId);
             modelBuilder.Entity<UserHistory>().HasOne(a => a.History).WithMany().HasForeignKey(a => a.HistoryId);
+
+            modelBuilder.Entity<User>().Property(u => u.Username).HasColumnName("name");
```

Comment:
you can remove this line now

Reply:
Done on my local changes. Will push when finished with clean up